index.js

```diff
@@ -0,0 +1,3 @@
+var a = require('./output/Examples');
+
+a.main();
```

Comment:
Oops, I think I generated this while trying to figure out the build. I'll have to remove it